### ISSUE TITLE
Cancel Screen Flashing When User Move Face Away

### ISSUE THREAD

**Issue by coolioxlr** [aws-amplify/amplify-ui-swift-liveness#113]
### Is your feature request related to a problem? Please describe.

We have a lot of users getting low liveliness scores because once the phone starts flashing the color light they move their face away. 

### Describe the solution you'd like

When flashing the color light and the user moves away their face cancels the flashing. 

### Describe alternatives you've considered

No

**Comment by phantumcode** [comment]
@coolioxlr Thanks for submitting your issue.  The user should be holding the face still when the color flashing lights appears in order to receive high confidence score.  Are you using the preview screen / hair check screen and providing end users with UX instructions to hold the face still in the oval?

**Comment by coolioxlr** [comment]
> @coolioxlr Thanks for submitting your issue. The user should be holding the face still when the color flashing lights appears in order to receive high confidence score. Are you using the preview screen / hair check screen and providing end users with UX instructions to hold the face still in the oval?

Yes, we are using the default screen and have the prompt to tell user to hold still. But we observed a lot of user start moving away when the screen start flashing so it will be ideal when the user move away to cancel the flashing.

**Comment by phantumcode** [comment]
@coolioxlr Thanks for your feedback.  We will add the request to our feature request backlog and provide updates here.

**Comment by thisisabhash** [comment]
This has been released in https://github.com/aws-amplify/amplify-ui-swift-liveness/releases/tag/1.4.6
